instance/models/mixins/openedx_storage.py

```diff
@@ -48,6 +48,13 @@ def swift_container_names(self):
         """
         return [self.swift_container_name]
 
+    @property
+    def s3_custom_domain(self):
```

Comment:
Can you move this to `/mixins/storage.py#S3BucketInstanceMixin`? There are a few other related properties there.